WcaOnRails/spec/factories/competitions.rb

```diff
@@ -6,9 +6,12 @@
     countryId "USA"
     information "Information!"
 
-    day = 1.year.ago
-    start_date day.strftime("%F")
-    end_date day.strftime("%F")
+    transient do
+      starting_at 1.year.ago
+    end
+
+    start_date { starting_at.strftime("%F") }
+    end_date { starting_at.strftime("%F") }
```

Comment:
Note: now we are able to easily set the competition's start date without caring about the 'strftime("%F")'.